cmd/mesh/testdata/manifest-generate/output/telemetry_override_kubernetes.yaml

```diff
@@ -1,3 +1,475 @@
+# Resources for Prometheus component
```

Comment:
This file wasn't modified in https://github.com/istio/operator/pull/286. What PR modified it in `master`?